retworkx-core/src/traversal/dfs_edges.rs

```diff
@@ -18,7 +18,25 @@ use petgraph::visit::{
     NodeIndexable, VisitMap, Visitable,
 };
 
-/// Return an edge list in depth first order.
+/// Return an edge list of the tree edges from a depth-first traversal.
+///
+/// The pseudo-code for the DFS algorithm is listed below. The output
+/// contains the tree edges found by the procedure.
+///
+/// ```norust
+/// DFS(G, v) is
+///     let S be a stack
+///     PUSH(S, (v, iterator of G.edges(v)))
+///     while (S != Ø)
+///         let (v, iterator) := LAST(S)
+///         if hasNext(iterator) then
+///             w := next(iterator)
+///             if w is not labeled as discovered then
+///                 label w as discovered                   # (u, w) is a tree edge
```

Comment:
Sorry, I made a type before:
```suggestion
///                 label w as discovered                   # (v, w) is a tree edge
```